src/views/blocks/rsvp/form/attendee-meta.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Block: RSVP
+ * Attendee Meta
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/blocks/rsvp/form/email.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
```

Comment:
TYPO: it's `ARTICLE`, not `ARTCILE`

Reply:
Although, if we're intending to do a search&replace later, it should get corrected...

Reply:
hah this was a copy paste from another template... will do a search replace